packages/manager/apps/dedicated/client/app/billing/payment/method/add/index.html

```diff
@@ -1,6 +1,19 @@
 <div class="billing-payment-method-add">
     <div data-ovh-alert="billing_payment_method_add_alert"></div>
 
+    <oui-message class="mb-3" type="warning">
```

Comment:
Should these messages come up only for US customers?

Reply:
I know the ticket is created based on US manager demand. But it concerns issue with ad blockers, which can be also applied to all regions.